src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -179,7 +180,11 @@ public void waitForVideoToEnd(final long timeout) {
   }
 
   private void clickElement(final String selector) {
-    wait.forElementClickable(By.cssSelector(selector)).click();
+    Actions builder = new Actions(driver);
+    builder.contextClick(wait.forElementClickable(By.cssSelector(selector)))
```

Comment:
How about caching results of `By.cssSelector(selector)` and then re-using it in next lines?